resources/assets/js/app/views/messages/messageDialogLayout.js

```diff
@@ -29,5 +31,18 @@ module.exports = Marionette.LayoutView.extend({
     },
     onBeforeDestroy: function () {
         this.collection.stopListening();
-    }
+    },
+
+    onDestroy: function () {
+        clearInterval(this.datesUpdateInterval);
+    },
+    onShow: function () {
+        this.datesUpdateInterval
+    },
+    datesUpdateInterval: setInterval(function() {
```

Comment:
this part will be executed during module init, so interval will start working before even this view will be initialized